backend/tests/integration/orcid/test_orcid_domain_models.py

```diff
@@ -539,3 +541,23 @@ def test_stale_record_db_version(self):
             domain_models.OrcidPusher(
                 self.orcid, self.recid, self.oauth_token, record_db_version=10
             )
+
+
+@pytest.mark.usefixtures("inspire_app")
+class TestOrcidPusherRecordRedirected(TestOrcidPusherBase):
+    def setup(self):
+        # redirected_record = TestRecordMetadata.create_from_file(
+        #     __name__, "test_orcid_domain_models_TestOrcidPusher.json"
+        # )
```

Comment:
do we need that? :)

Reply:
:D Yes, it's extremely important ;P
Thanks for noticing ;)